apps/web/src/components/icons/icon-base.tsx

```diff
@@ -1,16 +1,24 @@
 import * as React from "react";
 
-export interface IconProps extends React.ComponentPropsWithoutRef<"svg"> {
+export type IconElement = React.ElementRef<"svg">;
```

Comment:
nitpick but i wouldn't export types just for the sake of exporting unless we are actually using them somewhere else